cms/templates/maintenance/base.html

```diff
@@ -0,0 +1,21 @@
+<%page expression_filter="h"/>
+<%inherit file="../base.html" />
+<%def name='online_help_token()'><% return 'maintenance' %></%def>
+<%namespace name='static' file='../static_content.html'/>
+<%!
+from django.core.urlresolvers import reverse
+from django.utils.translation import ugettext as _
+%>
+<%block name="content">
+<div class="wrapper-content wrapper">
+    <div class="maintenance-header">
+        <h2>
+            <a href="${reverse('maintenance:maintenance')}">
```

Comment:
Is there a reason why this appears as a link while you are on the maintenance dashboard?